frontend/src/models/workspace.js

```diff
@@ -238,6 +238,14 @@ const Workspace = {
       .then((res) => res.ok)
       .catch(() => false);
   },
+  wipeAllVectorDbs: async function () {
+    return await fetch(`${API_BASE}/workspace/reset-all-vector-dbs`, {
+      method: "POST",
+      headers: baseHeaders(),
+    })
+      .then((res) => res.ok)
+      .catch(() => false);
+  },
```

Comment:
We should do this at the server level because this would not work if the provider is changed via API or if for some reason this request does not send.

Can we use a `postUpdate` hook in `updateENV` on the LLM_PROVIDER or embedding model key and check if the key changed and run this function for cleanup then?